tests/unit/test_disco_elb.py

```diff
@@ -657,10 +655,40 @@ def test_create_target_group_with_port_config(self):
             VpcId=TEST_VPC_ID,
             HealthCheckProtocol="HTTP",
             HealthCheckPort="80",
-            HealthCheckEnabled=True,
             HealthCheckPath="/"
         )
 
+    def test_create_target_group_tcp(self):
+        """Test creating a group non http/https"""
+        self.elb2.describe_target_groups.side_effect = EC2ResponseError(
+            status="mockstatus",
+            reason="mockreason"
+        )
+
+        instance_protocols = ('TCP',)
+        instance_ports = (80,)
+        elb_protocols = ('TCP',)
+        elb_ports = (80,)
+
+        self.disco_elb.get_or_create_target_group(
+            group_name="target_group_name",
+            vpc_id=TEST_VPC_ID,
+            port_config=DiscoELBPortConfig(
+                [
+                    DiscoELBPortMapping(internal_port, internal_protocol, external_port, external_protocol)
+                    for (internal_port, internal_protocol), (external_port, external_protocol) in zip(
+                        zip(instance_ports, instance_protocols), zip(elb_ports, elb_protocols))
+                ]
+            )
+        )
+        self.elb2.create_target_group.assert_called_with(
```

Comment:
this test shows that if a instance has a protocol that is TCP it will not call create target group with a health check path or a health check port since those values aren't supported

Reply:
I think only path is not supported. Port is supported and it actually will check if target responds on port e.g. `22`.